samples/Bluetooth/category.txt

```diff
@@ -1 +1,2 @@
 esp32
+
```

Comment:
Question: do you want a specific "Bluetooth" entry in the sample directory?
If yes, then we can do that to give more visibility.
In that case, add a very simple category file like this one with bluetooth inside and place it in the project directory. You will have to add as well a README.md file with the title (the # at first line) with a proper description (that what is showing up in the list). I guess copy/paste of the elements from the main readme should be enough. 
Let me know and happy to adjust the tool

Reply:
I agree that a Bluetooth section would be a great way to stand out these and increase the visibility! Let's have that.

Reply:
Let me work on that then :-) Most likely in next few days!

Reply:
Well, done, I used a break for that :-)